python/ray/serve/tests/test_metrics.py

```diff
@@ -944,10 +944,9 @@ def first_request_executing(request_future) -> bool:
     )
     print("ray_serve_deployment_queued_queries updated successfully.")
 
-    # TODO (shrekris-anyscale): This should be 0 once async task cancellation
```

Comment:
@shrekris-anyscale I love it when a plan comes together :)

Reply:
Nice!